setuptools_git_versioning.py

```diff
@@ -109,6 +112,8 @@ def parse_config(dist, _, value):  # type: (Distribution, Any, Any) -> None
     version_callback = value.get('version_callback', None)
     version_file = value.get('version_file', None)
     count_commits_from_version_file = value.get('count_commits_from_version_file', False)
+    global run_in_dir
```

Comment:
Please do not use global variables to store anything that is depending on the context. The interpreter will use the same value for both parent and child packages, and this will definitely lead to some unexpected errors.
It should be passed as an argument for all the functions which are calling `_exec`.